x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -65,14 +67,14 @@ const AppRoot: React.FunctionComponent<RouterProps> = React.memo(({ basename, st
               );
             }}
           />
-          <Route path="/alerts" component={AlertIndex} />
+          <Route path="/alerts" render={() => <AlertIndex />} />
```

Comment:
switched for consistency purposes and to be able to pass props down if/when needed

Reply:
why?

Reply:
@kqualters-elastic needs to pass down props and we can keep consistency with the rest of the app